app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,61 @@
+class VotesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_votable
+  before_action :ensure_not_author
+
+  def upvote
+    vote(+1)
+  end
+
+  def downvote
+    vote(-1)
+  end
```

Comment:
I'd prefer 1 endpoint for voting, and frontend will responsible for sending value. But this case also works)